test/D/di/Image/SConstruct_template

```diff
@@ -0,0 +1,15 @@
+# -*- mode:python; coding:utf-8; -*-
+
+import os
+import SCons.Node
+
```

Comment:
Please add the following. It instructs SCons to not initialize any tools in the DefaultEnvironment(), which will speed up the test (especially on windows)

`DefaultEnvironment(tools=[])`